package.json

```diff
@@ -114,6 +114,7 @@
     "@types/dompurify": "^3.0.5",
     "@types/emoji-mart": "^3.0.14",
     "@types/eslint": "^8.56.6",
+    "@types/lodash": "^4.17.0",
```

Comment:
@bludnic 
Do we really want lodash in our project?

Reply:
It was already there. These are just TS types.

Reply:
Maybe we will remove lodash in the future. Currently is not possible because we already use it.